src/easydmp/dmpt/api/v2/views.py

```diff
@@ -20,6 +31,30 @@
 from .serializers import *
 
 
+def get_template_export_from_url(url):
+    try:
+        with requests.get(url, timeout=0.1) as response:
+            response.raise_for_status()
+            return deserialize_template_export(response.content)
+        return {}
+    except requests.exception.Timeout:
+        raise ServiceUnavailable(
+            detail=f'Could not access {url}, timeout'
+        )
+
+
+class DRFIntegrityError(APIException):
+    status_code = 409
+    default_detail = 'Could not save changes.'
+    default_code = 'database_integrity'
+
+
+class ServiceUnavailable(APIException):
```

Comment:
This I would definitely move, for reuse. exceptions.py for example, or drf_exeptions.py.